tensilelite/Tensile/Components/LocalRead.py

```diff
@@ -219,7 +219,8 @@ def __call__(self, writer, kernel, bufferIdx, iui, epsi, tP):
             highBits = 0
             for tIdx in range(0, numberMTilesPerWave):
                 offset_val = (tP["localReadOffset"]+MIWaveGroupShape[tile01]*tIdx) * tP["bpeDS"]
-                if (kernel["LdsBlockSizePerPad%s"%tc] != 0) and (kernel["LdsPad%s"%tc] != 0):
+                # Disable this calculation for DTL, since we are only doing padding with M0
+                if (kernel["LdsBlockSizePerPad%s"%tc] != 0) and (kernel["LdsPad%s"%tc] != 0) and not kernel["DirectToLds%s"%tc]:
```

Comment:
Do we really need this condition?
DTL is related to write only.
For local read, it should not matter if it is DTL or non DTL.

Reply:
Ah you're right, the DTL check is not actually needed, removed.